### ISSUE TITLE
What to do if OGIP conventions and the FITS standard are in conflict?

### ISSUE THREAD

**Issue by maxnoe** [open-gamma-ray-astro/gamma-astro-data-formats#191]
I just noticed a conflict between the OGIP conventions and the FITS standard, which is explicitly commented on in the FITS time paper, which definitions were taken over for the FITS standard version 4.0.

Specifically, in this instance, it is the keyword and its possible values specifying the time reference location.

In OGIP, it is `TIMEREF` with value `'LOCAL'`, which is given in GADF (but, pointing incorrectly to the FITS standard) and in the FITS time paper and the current standard it is `TREFPOS` with multiple possible values and `'TOPOCENTER'` being the equivalent to `'LOCAL'`.

While reworking the time definitions is also needed with respect to the current standard, I think we should take a decision on how to resolve conflicts between FITS *standard* and OGIP *convention*.

My personal preference would be to follow the FITS standard.

**Comment by jknodlseder** [comment]
Maybe check what other instruments do as this at the end corresponds most to what users (and software) expects.

**Comment by maxnoe** [comment]
I don't think we should let the legacy behavior of other instruments or software take precedence over the current version of the FITS standard for new iterations of the specification. 

FERMI for example uses `TIMEREF` and the now deprecated `RADECSYS`. But FERMI software mostly precedes the current FITS standard (4.0 from August 2018) and probably even the FITS time paper (2015).

I think we should take the decision to give the FITS standard precedence in these kinds of cases, so we are in agreement with the current version and not perpetuate legacy behavior.

**Comment by PaulKuin** [comment]
Historically, FITS was developed to interchange data on tape, mostly from the Radio and space community (from IUE to CGRO). The adopted convention is to make all changes such that they do not break the older FITS versions. As a result, there were extensions to the FITS from both OGIP and CDS which differed. It is allowed to differ, for example, the Swift headers specify time in a way that is discouraged nowadays, but in 2004 was OK. So I would not worry too much about the above. As long as you don't break the backwards compatibility it is OK.
